receiver/sqlserverreceiver/metadata.yaml

```
@@ -199,6 +203,38 @@ metrics:
       value_type: double
     attributes: [physical_filename, logical_filename, file_type]
     extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.database.io.write_latency:
+    enabled: false
+    description: Total time that the users waited for writes.
+    unit: "s"
+    sum:
+      monotonic: true
+      aggregation_temporality: cumulative
+      value_type: double
+    attributes: [ physical_filename, logical_filename, file_type ]
+    extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.database.operations:
+    enabled: false
+    description: The number of operations issued on the file.
+    unit: "{operations}"
+    sum:
+      monotonic: true
+      aggregation_temporality: cumulative
+      value_type: int
+      input_type: string
+    attributes: [ physical_filename, logical_filename, file_type, direction ]
+    extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.database.io:
+    enabled: false
+    description: The number of bytes of I/O on this file.
+    unit: "{By}"
```

We don't need the brackets here
```suggestion
    unit: "By"
```

---

Done, thanks for pointing this out!